fe/fe-core/src/main/java/org/apache/doris/statistics/Statistics.java

```diff
@@ -129,7 +129,7 @@ public double computeSize() {
     }
 
     public double dataSizeFactor() {
-        return computeTupleSize() / K_BYTES;
+        return Math.max(computeTupleSize() / K_BYTES, 1);
```

Comment:
please add explaination for this change and list the tpcds 1T perf data comparison, some cases can be put into consideration, such as q4/q11.